src/styles.css

```diff
@@ -167,7 +167,7 @@
 /* Image */
 .cio-question-image {
   width: 100%;
-  height: 100%;
+  max-height: 100%;
```

Comment:
This was causing an issue with the variation for Big table. Remove it and you'll be able to reproduce the css bug

Reply:
I tried it but couldn't reproduce, do you mind posting an example?